test/testutils.jl

```diff
@@ -28,8 +28,9 @@ end
 # testing the implementation of a discrete univariate distribution
 #
 function test_distr(distr::DiscreteUnivariateDistribution, n::Int;
-                    testquan::Bool=true)
-
+                    testquan::Bool=true, rng::AbstractRNG=MersenneTwister(),
+                    test_scalar_rand::Bool=false)
```

Comment:
I'd like to keep the changes a bit more minimal - I think we can continue working with the `MersenneTwister` and test the the scalar `rand` method by default?
```suggestion
                    testquan::Bool=true)
```